src/plugins/auto/executable_network.cpp

```diff
@@ -206,6 +206,9 @@ MultiDeviceExecutableNetwork::MultiDeviceExecutableNetwork(const std::string&
              _loadContext[i].future =  _loadContext[i].promise.get_future();
               auto* contextPtr = &_loadContext[i];
              _loadContext[i].task = [this, contextPtr, modelPath, network]() mutable {
+                      // check auto batching for GPU only, do not affect CPU load efficiency
+                      if (contextPtr->deviceInfo.deviceName.find("GPU") != std::string::npos)
```

Comment:
per discussion in the original Auto-Batching PR, (https://github.com/openvinotoolkit/openvino/pull/7883#discussion_r773392528) it is better to check the support of OPTIMAL_BATCH_SZIE, and if it is supported and the value for the network is >1, then apply the auto-batching

Reply:
add the pre-requisite of optimal_batch_size >1